src/inmanta/plugins.py

```diff
@@ -213,7 +213,12 @@ def __init__(self, namespace: Namespace) -> None:
 
         filename: Optional[str] = inspect.getsourcefile(self.__class__.__function__)
         assert filename is not None
-        line: int = inspect.getsourcelines(self.__class__.__function__)[1] + 1
+        try:
+            line: int = inspect.getsourcelines(self.__class__.__function__)[1] + 1
+        except OSError:
+            # In case of bytecompiled code there is no source line
+            line = 1
```

Comment:
```suggestion
            line = -1
```
I'm not completely sure, this might break tests, but I believe we already use `-1` in some places where line number does not make sense, e.g. [here](https://github.com/inmanta/inmanta-core/blob/5eb1419d2e142c8f9bb8ba2a28721801c94711dd/src/inmanta/module.py#L1648).

Reply:
That does not work. It excepts here: https://github.com/inmanta/inmanta-core/blob/master/src/inmanta/ast/__init__.py#L155